packages/widgetbook/README.md

```diff
@@ -38,19 +52,30 @@ class HotReload extends StatelessWidget {
 }
 ```
 
-### Running the Widgetbook
-
-To run the `Widgetbook` create a new main method, e.g. in `stories/main.dart`:
+In the `stories/main.dart` run the `HotReload` widget:
 
 ```dart
 void main() {
   runApp(HotReload());
 }
 ```
 
+## Run the Widgetbook
+
+`Widgetbook` is supported on the following environments.
+
+Environment | Status | Annotation
+------------ | ------------- | ------------- 
+MacOS | ✅ | 
+Windows | ✅ |
+Web | ✅ | No hot reload, but hot restart (see [Issue 4](https://github.com/widgetbook/widgetbook/issues/4))
+iOS and Mobile | ➖ | Will run, but is not optimized. If you see a usecase for `Widgetbook` on mobile let us know. 
+
+See the [Desktop support for Flutter](https://flutter.dev/desktop) page for setup instructions.
+
 Run the `Widgetbook` main method by executing `flutter run -t stories/main.dart`.
 
-### Inject your widgets
+# Inject your widgets
 
 Your widgets can be organized into different areas of interest by using `Category`, `Folder`, `WidgetElement` and `Story`.
```

Comment:
Would you like to explain what the different areas of interest do?

Reply:
But in my opinion, it is already sufficient since we'll show an example directly afterwards